samples/wifi/sta/Kconfig

```diff
@@ -8,6 +8,14 @@ source "Kconfig.zephyr"
 
 menu "Nordic Sta sample"
 
+config MAXIMUM_ITERATIONS
```

Comment:
This doesn't look like a user facing entry, its more of an internal state. This should either be renamed (connection_timeout) or removed from Kconfig.